mozilla-release/mobile/android/base/java/org/mozilla/gecko/controlcenter/OverviewFragment.java

```diff
@@ -352,27 +359,29 @@ public void refreshUI() {
 
     @Override
     public void onClick(View v) {
-        switch (v.getId()) {
-            case R.id.cc_ghostery_trust_site_button:
-                handleTrustButtonClick();
-                break;
-            case R.id.cc_ghostery_restrict_site_button:
-                handleRestrictButtonClick();
-                break;
-            case R.id.cc_ghostery_pause_button:
-                handlePauseButtonClick();
-                break;
-            case R.id.cc_enhanced_tracking_info_button:
-                handleTrackingInfoButtonClick();
-                break;
-            case R.id.cc_enhanced_blocking_info_button:
-                handleAdBlockingInfoButtonClick();
-                break;
-            case R.id.cc_smart_blocking_info_button:
-                handleSmartBlockingInfoButtonClick();
-                break;
-            case R.id.cc_close_info_view_button:
-                handleInfoViewCloseButtonClick();
+        if (controlCenterSettingsData != null) {
```

Comment:
I'll prefer a `return` (three lines added instead of 21 removed and 23 added).
```java
if (controlCenterSettingsData == null) {
    return;
}
```